test/spec/provider/cloud-element-templates/cmd/ChangeElementTemplateHandler.spec.js

```diff
@@ -640,6 +640,69 @@ describe('cloud-element-templates - ChangeElementTemplateHandler', function() {
 
     });
 
+
+    describe('update task type', function() {
```

Comment:
Shouldn't we have test cases for

* [new and old specified](https://github.com/bpmn-io/bpmn-js-properties-panel/blob/4f0248a4d0ee27675a2202d05c8d777803f35033/test/spec/provider/element-templates/cmd/ChangeElementTemplateHandler.spec.js#L1669)
* [no new template specified](https://github.com/bpmn-io/bpmn-js-properties-panel/blob/4f0248a4d0ee27675a2202d05c8d777803f35033/test/spec/provider/element-templates/cmd/ChangeElementTemplateHandler.spec.js#L4271)